src/PrestaShopBundle/Install/Database.php

```diff
@@ -104,6 +104,15 @@ public function testDatabaseSettings($server, $database, $login, $password, $pre
             }
         }
 
+        if (!$errors) {
+            if (($select_error = Db::checkSelectPrivilege($server, $login, $password, $database, $prefix)) !== true) {
+                $errors[] = $this->translator->trans('You must be granted the privilege to select data in the tables of database "%s". Ask your hosting provider to enable it.', ['%database%' => $database], 'Install');
+                if ($select_error != false) {
```

Comment:
I think you can check `$select_error` with `!==` here, since `Db::checkSelectPrivilege` returns a boolean or a string.

Reply:
:heavy_check_mark: